.travis.yml

```diff
@@ -0,0 +1,49 @@
+# You must sign into travis-ci.org and set the commit hook on your project for travis to
+# run on your project. The secure: variable must be generated by running `travis encrypt`
+# on a github oauth key that you can generate using curl.
```

Comment:
Can you provide a link to the instructions for generating the GitHub OAuth key using `curl`?